hadoop-ozone/ozone-recon/src/main/java/org/apache/hadoop/ozone/recon/ReconUtils.java

```diff
@@ -90,7 +90,7 @@ public static File getReconDbDir(Configuration conf, String dirConfigKey) {
    * @param destPath destination path to untar to.
    * @throws IOException ioException
    */
-  public static void untarCheckpointFile(File tarFile, Path destPath)
+  public void untarCheckpointFile(File tarFile, Path destPath)
```

Comment:
Yes I agree. But we decided to make the ReconUtils non static so that we can inject a mock instance of it and unit test cleanly, removing the need for Powermocks.